src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
```

Comment:
Change the call of the method in Main.java to Parser.method() because it is static. (Line 84)